src/main/java/org/example/homework9/MobilePhone.java

```diff
@@ -0,0 +1,86 @@
+package org.example.homework9;
+
+import java.util.ArrayList;
+
+public class MobilePhone {
+    private String myNumber;
+    private ArrayList<Contact> myContacts;
+
+    public MobilePhone(String myNumber, ArrayList<Contact> myContacts) {
+        this.myNumber = myNumber;
+        this.myContacts = myContacts;
```

Comment:
```suggestion
        this.myContacts = new ArrayList<>();
```
This is not something critical, when you leave just one parameter in constructor it means, that you creating a new phone instance without providing a list of contacts(it is initialized as empty). As for me it fits better for our task